libs/docusaurus/src/schematics/app/files/docusaurus.config.js.template

```diff
@@ -83,10 +84,19 @@ module.exports = {
       '@docusaurus/preset-classic',
       {
         docs: {
+          // It is recommended to set document id as docs home page (`docs/` path).
+          homePageId: 'doc1',
```

Comment:
I'm concerned about this property because when I remove it and navigate to `/docs` I get a 404. How will this effect existing users that migrate?

Reply:
I tested this against a 0.5.0 generated application and all of the docs worked. This is because the items in older versions have explicit links i.e.
```
items: [{ to: 'docs/doc1'}, ...] vs items: [{ to: 'docs'}, ...] vs 
```
Looking at current master for docusaurus, it looks like `homePageId` has already been [deprecated](https://github.com/facebook/docusaurus/commit/0a33a5230108bb5fde0570882fdc328d2148a0af), so if there is no change in behavior then I think I'll remove it.